### DISCUSSION TITLE
RelateConfig GORMTag Wrong Type in Documentation

### DISCUSSION THREAD

**Discussion by hasan-kesserwan** [go-gorm/gorm#6580] [Q&A]
Hello 👋 
First of all thank you for the amazing tool you created, it helped me a lot.
One thing just confuses me that in the gen tool documentation to define an association in the GenerateModel function it goes like this:
`card := g.GenerateModel("credit_cards")
customer := g.GenerateModel("customers", gen.FieldRelate(field.HasMany, "CreditCards", card, 
    &field.RelateConfig{
        // RelateSlice: true,
        GORMTag: "foreignKey:CustomerRefer",
    }),
)

g.ApplyBasic(card, custormer)`

but when trying to that in my code it's showing that the GORMTag field is not a string, it is of type GORMTag which is map[string][]string, can you please provide me with an example I can relate to to define an association here, thank you.

**Comment by afeish** [comment]
use the following code:

```go
	gen.FieldRelate(field.HasMany, "CreditCards", card, &field.RelateConfig{
                       // RelateSlice: true,
			GORMTag:            field.GormTag{"foreignKey": []string{"CustomerRefer"}},
		}),
```
